profiles/desc/llvm_targets.desc

```diff
@@ -25,4 +25,5 @@ VE - NEC SX-Aurora TSUBASA vector engine target
 WebAssembly - WebAssembly backend
 X86 - X86 CPU target (includes amd64)
 XCore - XCore CPU target
+XE - Intel Xe (GPU) target
```

Comment:
It's weird to list stuff that's not actually an LLVM target (as in, it's not in LLVM) here. Why not make it a regular `xe` flag on ispc?